unittests/test_scotty.py

```diff
@@ -12,7 +12,32 @@
 from scottypy.scotty import CombadgeRust, CombadgePython
 
 
-def combadge(request, context):
+class APICallLogger:
+    def __init__(self):
+        self.calls = []
+
+    def log_call(self, request):
+        try:
+            json = request.json()
+        except TypeError:
+            json = None
+        self.calls.append(dict(url=request.url, params=request.qs, json=json))
+
+    @contextlib.contextmanager
+    def isolate(self):
+        old_calls = self.calls
+        self.calls = []
+        yield
+        self.calls = old_calls
+
+
+def one_or_raise(items):
```

Comment:
Let's rename to `_one_or_raise` (internal helper func).

Reply:
Also, maybe make in a method of `APICallLogger`? Thus its purpose can be clearer.
It's always called in the same way: `assert one_or_raise(api_call_logger.calls) .....`